### PR TITLE
Collect additional telemetry for crypto/runtime errors

### PR DESCRIPTION
**Pull request by tnorling**
- Adds additional check for the subtle property on crypto
- Fixes error stack logic to keep top 5 lines instead of bottom 5
- Collects common, known runtime errors and redacts when necessary

### CODE REVIEW

**Review by konstantin-msft — APPROVED** [review]

File: lib/msal-common/src/telemetry/performance/PerformanceClient.ts [hunk 1]
```diff
@@ -198,22 +198,39 @@ export function compactStack(stack: string, stackMaxSize: number): string[] {
...
+    const firstLine = stackArr[0];
+    if (
+        firstLine.startsWith("TypeError: Cannot read property") ||
+        firstLine.startsWith("TypeError: Cannot read properties of") ||
+        firstLine.startsWith("TypeError: Cannot set property") ||
+        firstLine.startsWith("TypeError: Cannot set properties of") ||
+        firstLine.endsWith("is not a function")
     ) {
-        const line = stackArr[ix];
+        // These types of errors are not at risk of leaking PII. They will indicate unavailable APIs
```
Comment (sameerag) [inline lib/msal-common/src/telemetry/performance/PerformanceClient.ts:213]: Can we guarantee this?

**Review by sameerag — APPROVED** [review]

File: lib/msal-common/src/telemetry/performance/PerformanceClient.ts (hunk 1 above)
Comment (tnorling) [reply lib/msal-common/src/telemetry/performance/PerformanceClient.ts:213]: Yes, these are thrown when we call an API or access an object property that doesn't exist, neither of which are PII and we control this.

Comment (sameerag) [reply lib/msal-common/src/telemetry/performance/PerformanceClient.ts:213]: *can we get a guidance to the team so folks don't accidentally add PII?

Comment (tnorling) [reply lib/msal-common/src/telemetry/performance/PerformanceClient.ts:213]: I'm saying there's not a risk of that. APIs and object properties are not PII

**Pull request merged**